resources/js/components/Crud/Fields/Modal/FieldModal.vue

```diff
@@ -112,6 +115,12 @@ export default {
         cancel() {
             this.$bvModal.hide(this.modalId);
         },
+        setId() {
+            this.modalId = Lit.uuid();
```

Comment:
@jannescb have you tried this? I'm not sure whether this method exists for `Lit`. There somewhere is a uuid implemented I'm just not sure where at the moment.

Reply:
I Added a global method here:

https://github.com/litstack/litstack/blob/7e374b08b3656d6230bfe646722a86bead696fee/resources/js/common/lit.js#L120-L129